sigstore-java/src/main/java/dev/sigstore/json/canonicalizer/JsonCanonicalizer.java

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright 2022 The Sigstore Authors.
```

Comment:
I believe you need to keep the original authors in and add a special case in our license handler. like: 
- https://github.com/sigstore/sigstore-java/blob/main/sigstore-java/build.gradle.kts#L92-L95
- https://github.com/sigstore/sigstore-java/blob/main/sigstore-java/src/main/java/dev/sigstore/encryption/certificates/transparency/CTConstants.java#L3

Reply:
thanks for the links! fixed.